libobs/obs-source.c

```diff
@@ -1638,6 +1648,23 @@ static inline bool set_planar444_sizes(struct obs_source *source,
 	return true;
 }
 
+static inline bool
+set_planar444_16le_sizes(struct obs_source *source,
```

Comment:
Don't need `le` in function name. Same implementation even if `be`. Applies to all three new functions.